EESSI-install-software.sh

```diff
@@ -187,6 +187,22 @@ fi
 # assume there's only one diff file that corresponds to the PR patch file
 pr_diff=$(ls [0-9]*.diff | head -1)
 
+# install any additional required scripts
+# order is important: these are needed to install a full CUDA SDK in host_injections
+install_scripts_changed=$(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^install_scripts.sh$' > /dev/null; echo $?)
+if [ ${install_scripts_changed} == '0' ]; then
+    # for now, this just reinstalls all scripts. Note the most elegant, but works
+    ${TOPDIR}/install_scripts.sh --prefix ${EESSI_CVMFS_REPO}
+fi
+
+# Install full CUDA SDK in host_injections
+# Hardcode this for now, see if it works
+# TODO: We should make a nice yaml and loop over all CUDA versions in that yaml to figure out what to install
+${EESSI_CVMFS_REPO}/gpu_support/nvidia/install_cuda_host_injections.sh 12.1.1
```

Comment:
Wow, I really messed up here...the name of the script and arguments are incorrect!